### ISSUE TITLE
Plugin GPU Monitor wont show itsself in IDE

### ISSUE THREAD

**Issue by Kokonut133** [andrewpeng02/gpu-monitor-plugin#1]
I have installed the Plugin https://plugins.jetbrains.com/plugin/13597-gpu-monitor via the IDE Plugin downloader and restarted the IDE. This yields GPU Monitor being reported as "ON" when searched via the top right search field and it being visible in my settings. Yet there is no indication in the IDE how to see or access it. I have tried Run and Debug additionally, with nothing revealing itsself.

I would like to use this Plugin, but it is invisible to me. If there is any additional setup required, I wouldn't know where to go.

**Comment by andrewpeng02** [comment]
There should be a button in the lower right hand side (part of the toolbar), which you can click on. Which jetbrains product and version are you using?
![Annotation 2020-04-02 093035](https://user-images.githubusercontent.com/47485510/78261355-c245fc80-74c4-11ea-90ab-a3b5c6d372a0.png)

**Comment by Kokonut133** [comment]
Hi. I do not have such a symbol, even so it shows me it is ON. Pycharm 2019.3.1 Community Edition
![Annotation 2020-04-04 153917](https://user-images.githubusercontent.com/30372971/78452282-b86df600-768a-11ea-90bf-70ab021ef87a.png)

**Comment by andrewpeng02** [comment]
The GPU monitor button is in a tool window. You can try to access it by going into View -> Tool Windows -> GPU Monitor. To enable the right side tool window, either try to click on the button on the bottom left that looks like a stack of papers, or make sure that View -> Appearance -> Tool Window Bars is enabled.

**Comment by Kokonut133** [comment]
Hi. I have checked both ways but found not such an option. see below. I also checked settings -> menu and toolbars and looked through for any indication of the GPU monitor which wasn't there.
![Annotation 2020-04-06 114946](https://user-images.githubusercontent.com/30372971/78546317-7a451380-77fd-11ea-9a2d-926db2fe13b4.png)
![Annotation 2020-04-04 153917](https://user-images.githubusercontent.com/30372971/78546318-7addaa00-77fd-11ea-9eb0-e5cc61929f18.png)
